### ISSUE TITLE
Update for HPOS

### ISSUE THREAD

**Issue by mateusz-holownia** [alexmustin/woocustomemails#39]
Is it possible to update the plugin with the new woocommerce HPOS functionality?

This plugin is incompatible with the "High Performance Order Storage (HPOS)" 

DOC:
[High-Performance-Order-Storage-Upgrade](https://github.com/woocommerce/woocommerce/wiki/High-Performance-Order-Storage-Upgrade-Recipe-Book)

Will the plugin return to the WordPress repository?

**Comment by Frits1980** [comment]
Does anyone work at this project?

**Comment by alexmustin** [comment]
> Does anyone work at this project?

@Frits1980 Yes I receive all the notifications for these requests, but I also have a very busy schedule with paid projects.

Would you like to contribute your own code updates to the project? Please let me know!

**Comment by Frits1980** [comment]
If I could write code I would. I can only read it a little. The only thing I can do is pay you for your work.

**Comment by tomrobak** [comment]
I did a fork for HPOS - plugin is not using wc tables, so simple declaration is enough

**Comment by BE-Zizoo** [comment]
New updated fork with HPOS available here : https://github.com/BE-Zizoo/woocustomemails

**Comment by alexmustin** [comment]
@BE-Zizoo this repo is no longer maintained, as I have created a new product:
[https://github.com/alexmustin/smart-product-emails](https://github.com/alexmustin/smart-product-emails)

Thanks!

**Comment by BE-Zizoo** [comment]
Great ! Good luck ! When i use the new product, will it import the existing templates from this product ?
